src/components/MdnRequest.vue

```diff
@@ -1,11 +1,18 @@
 <template>
-	<div v-if="hasMdnRequest" id="mail-message-has-mdn-request">
-		{{
-			t('mail', 'The sender of this message has asked to be notified when you read this message.')
-		}}
-		<button @click="sendMdn">
-			{{ t('mail', 'Notify the sender') }}
-		</button>
+	<div v-if="hasMdnRequest || mdnSent" id="mail-message-has-mdn-request">
+		<span v-if="hasMdnRequest">
+			{{
+				t('mail', 'The sender of this message has asked to be notified when you read this message.')
+			}}
+			<button @click="sendMdn">
```

Comment:
it might be good if we disable the button as long as the async request is awaited in the background

Reply:
yep